File: backend/src/entities/SchoolHoliday.ts
import { Entity, PrimaryGeneratedColumn, Column } from 'typeorm';

@Entity('school_holidays')
export class SchoolHoliday {
  @PrimaryGeneratedColumn()
  id: number;

  @Column({ type: 'date' })
  date: Date;

  @Column()
  description: string;
}

File: backend/app/models/school_holiday.py
from sqlalchemy import Column, Integer, String, Date
from .base import Base

class SchoolHoliday(Base):
    __tablename__ = "school_holidays"
    
    id = Column(Integer, primary_key=True, index=True)
    date = Column(Date, nullable=False, unique=True, index=True)
    description = Column(String, nullable=False)